event_listener/dfusion_db/abis/SnappBase.json

```diff
@@ -128,7 +151,8 @@
       ],
       "payable": false,
       "stateMutability": "view",
-      "type": "function"
+      "type": "function",
+      "signature": "0x8f32d59b"
```

Comment:
I don't think the ABI should have these signatures. This must be the result of a different type of compile.

Reply:
I resolved it